Tools/MGCB/CommandLineParser.cs

```diff
@@ -284,6 +284,8 @@ private IEnumerable<string> Preprocess(IEnumerable<string> args)
                 if (arg.StartsWith("/@:") || arg.StartsWith("-@:"))
                 {
                     var file = arg.Substring(3);
+                    Directory.SetCurrentDirectory(Path.GetDirectoryName(file));
```

Comment:
I added the workingDir argument a while back so I could debug from MonoDevelop, which has no working directory option in its debug settings.

The usual mode of operation when running MGCB from the command line has been to change directory to the location of the .mgcb file, but setting the current directory from the location of the .mgcb file would make the debugging process easier in MonoDevelop (no need for the workingDir argument).